emantomo/protocols/protocol_tomo_extraction.py

```diff
@@ -190,10 +190,7 @@ def createOutputStep(self):
         self.outputSubTomogramsSet = self._createSetOfSubTomograms(self._getOutputSuffix(SetOfSubTomograms))
         self.outputSubTomogramsSet.setSamplingRate(self.getInputTomograms().getSamplingRate() / self.downFactor.get())
         self.outputSubTomogramsSet.setCoordinates3D(self.inputCoordinates)
-        acquisition = TomoAcquisition()
-        acquisition.setAngleMin(self.getInputTomograms().getFirstItem().getAcquisition().getAngleMin())
```

Comment:
Here, could we probably loop through each Tomogram in the SetOfTomograms and assign each Subtomogram the acquisition info corresponding to their Tomogram? We talked about removing the acquisition property from the SetOfTomograms so each Tomogram can have its own information, so it might be safer to assign the information that way.